### ISSUE TITLE
Run through target analysis locally

### ISSUE THREAD

**Issue by matthewfeickert** [Samcoodess/reana-dms#1]
As @pibion and @besk467 have identified https://github.com/Cisneros-Galaxy/RCFM as being a good target analysis for implementation it would be good for @Samcoodess to:

* [x] Make a public fork of the analysis repository
   - https://github.com/Samcoodess/RCFM
* [x] Verify all the data sources needed for the analysis
   - > The entire thing runs from th[e] repo alone! --- @pibion
* [x] Get the analysis running locally
   - [x] Create a Python `pip` `requirements.txt` or `conda/mamba` `evironment.yml` that defines all the requirements of the analysis computational environment
   - [x] Document all steps associated with this (could upstream instructions back to the target analysis repo). This documentation can be done in a "notes to self" style as you go, and a great place to do it is in a GitHub Issue (maybe even this one).

(Note the amount of data or compute required might prohibit doing this locally &mdash; I don't know. In this case, just getting a subset or examples running locally first is enough and we can then pivot to getting things setup to run on REANA.)

**Comment by matthewfeickert** [comment]
> Get the analysis running locally

@Samcoodess along these lines while you're getting setup on https://github.com/Samcoodess/RCFM as you're inevitability going to need to write some of your own code and notes, are you familiar with using a Pull Request (PR) based workflow for software development and doing all your work on "feature" branches (so that the `main`/`master` branch is only committed to via merges of PRs? If so, great. If not, no worries as I have some references that I can direct you to.

**Comment by Samcoodess** [comment]
>are you familiar with using a Pull Request (PR) based workflow for software development and doing all your work on "feature" branches (so that the `main`/`master` branch is only committed to via merges of PRs? 

  Yes, @matthewfeickert  I am familiar with the git feature branch workflow. I also reviewed this article and revised my knowledge.[Git feature branch workflow]( https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow )
Do you have better references that can be useful for my software development journey?

**Comment by matthewfeickert** [comment]
@Samcoodess In general Atlassian's writing on version control is good so I think you found a good resource and would encourage you to also work through their whole ["Getting Started" section](https://www.atlassian.com/git/tutorials/setting-up-a-repository) as the subsection on [Undoing changes](https://www.atlassian.com/git/tutorials/undoing-changes) and [Rewriting history](https://www.atlassian.com/git/tutorials/rewriting-history) are quite important to learn in general (interactive `rebase` and `reflog` are core parts of my daily workflow). :+1: (You may have already done this thoug).

I think beyond that I would also at some point read through [Version Control](https://henryiii.github.io/se-for-sci/content/week6/git.html) section of @henryiii's [Software Engineering for Scientific Computing](https://henryiii.github.io/se-for-sci/) course.
